packages/design-system/src/components/N8nMenu/Menu.vue

```diff
@@ -133,11 +133,18 @@ const onSelect = (item: IMenuItem): void => {
 	background-color: var(--menu-background, var(--color-background-xlight));
 }
 
+.menuHeader {
+	display: flex;
+	flex-direction: column;
+	flex: 1 1 auto;
+	overflow-y: auto;
```

Comment:
If the window size is too small, the project. list just disappears entirely and I'm not sure whether this is intentional. Should we add a `min-height` here?

Reply:
Could help until a certain point but If we hit around 768px height we need to rethink the layout under that resolution

Reply:
At 768px height we can see 3 projects but I think that's the max
Under that we have to make something else disappear

![CleanShot 2024-03-28 at 09 43 51](https://github.com/n8n-io/n8n/assets/5410822/01d6f1b4-e091-4e8e-97d2-4050d1b0175a)